Linux Script/configure-linux.sh

```diff
@@ -350,10 +353,15 @@ checkIfLogglyServersAccessible() {
   fi
 
   echo "INFO: Checking if '$LOGGLY_ACCOUNT' subdomain is valid."
-  if [ $(curl --head -s --request GET $LOGGLY_ACCOUNT_URL/login | grep "200 OK\|HTTP/2 200" | wc -l) ] >0; then
-    echo "INFO: $LOGGLY_ACCOUNT_URL is valid and reachable."
+  if [[ $LOGGLY_ACCOUNT != $INVALID_SUBDOMAIN ]]; then
```

Comment:
@mchaudhary, This condition is to check if the **subdomain** is not invalid i.e. it does not contain **.loggly.com** in it.